phoenix_liveview/phoenix_liveview.exs

```diff
@@ -0,0 +1,134 @@
+Application.put_env(:phoenix, :json_library, Jason)
```

Comment:
Required to avoid a warning

Reply:
fwiw, we plan to address it with Mix.install for the next Elixir release. We envision this API:

```elixir
Mix.install(
  [
    :phoenix
  ],
  config: [
    phoenix: [json_library: Jason]
  ]
)
```

we're gonna use the config as part of the cache key, if it changes, the cache is stale so everything is recompiled.